model_building/model_building.py

```diff
@@ -184,7 +192,11 @@ def process(self, campaign_configuration, regression_inputs, processes_number):
 
             if 'save_training_regressors' in campaign_configuration['General'] and campaign_configuration['General']['save_training_regressors']:
                 # Build the regressor trained on the train set only
-                training_regressor = regressor.Regressor(campaign_configuration, best_conf.get_regressor(), best_conf.get_x_columns(), regression_inputs.scalers)
+                if hasattr(best_conf, '_wrapped_experiment_configuration'):
```

Comment:
Given how often it appears, I'd create a `is_wrapper()` class method (or other names you like better) that performs this `hasattr()` call

Reply:
Added and modified